app/scripts/router.js

```diff
@@ -83,7 +83,8 @@ function (
         router: this,
         user: this.user,
         window: this.window,
-        screenName: this.fragmentToScreenName(Backbone.history.fragment)
+        screenName: this.fragmentToScreenName(Backbone.history.fragment),
+        model: this.formModel
```

Comment:
`model` is a generic attribute on a Backbone view and is automatically imported. I debated whether to use a new name, like `prefillModel`, but didn't for reasons that I forget. Need to think about this more.